### PR TITLE
specs-go: add missing urls field of descriptor

### PR DESCRIPTION
**Pull request by coolljt0725**
Signed-off-by: Lei Jitang <leijitang@huawei.com>

### CODE REVIEW

**Comment by wking** [comment]
ff1d1314 looks good to me.  The JSON Schema needs to be updated too,
but that can happen in a follow-up PR if you don't want to cover it
here.

**Comment by coolljt0725** [comment]
@wking  thanks for your reminder, will do that

**Comment by coolljt0725** [comment]
@wking  added a commit to update the JSON schema

**Comment by runcom** [comment]
LGTM, ping @vbatts @philips can we get this in so tooling can start using this?

**Comment by jonboulle** [comment]
commit message/title has a typo (filed -> field) and could be squashed with the other commit

**Comment by wking** [comment]
`manifestDescriptor` in `defs-image.json` should be updated too.

**Comment by coolljt0725** [comment]
@wking  @jonboulle 
The commit message and title has updated, also squash to a commit.
Add description field.
Update `manifestDescriptor` in `defs-image.json`

File: schema/content-descriptor.json [hunk 1]
```diff
@@ -14,6 +14,13 @@
     },
     "digest": {
       "$ref": "defs-image.json#definitions/digest"
+    },
+    "urls": {
+      "description": "a list of urls from which this object may be downloaded",
+      "type": "array",
+      "items": {
+        "type": "string"
```
Comment (wking) [inline schema/content-descriptor.json]: These should use the [`uri` format][1]:

```json
"type": "string",
"format": "uri"
```

[1]: https://tools.ietf.org/html/draft-wright-json-schema-validation-00#section-7.3.6

**Comment by coolljt0725** [comment]
@wking  updated per your suggestions

File: schema/content-descriptor.json [hunk 2]
```diff
@@ -14,6 +14,14 @@
     },
     "digest": {
       "$ref": "defs-image.json#definitions/digest"
+    },
+    "urls": {
+      "description": "a list of urls from which this object may be downloaded",
+      "type": "array",
+      "items": {
+        "type": "string",
+	"format": "uri"
```
Comment (wking) [inline schema/content-descriptor.json]: Inconsistent indent?

**Comment by coolljt0725** [comment]
@wking  updated

**Comment by wking** [comment]
Indents are still a bit off.  And we do have a rule to fix them for you:

  $ git describe --always
  v1.0.0-rc1-99-g6975241
  $ make fmt
  $ git diff
  diff --git a/schema/defs-image.json b/schema/defs-image.json
  index 2a029ab..b0046a0 100644
  --- a/schema/defs-image.json
  +++ b/schema/defs-image.json
  @@ -32,13 +32,13 @@
           "digest": {
             "$ref": "#definitions/digest"
           },
  -       "urls": {
  +        "urls": {
             "description": "a list of urls from which this object may be downloaded",
             "type": "array",
             "items": {
  -           "type": "string",
  -           "format": "uri"
  -         }
  +            "type": "string",
  +            "format": "uri"
  +          }
           },
           "platform": {
             "id": "https://opencontainers.org/schema/image/platform",

**Comment by coolljt0725** [comment]
@wking  updated, `make fmt`  to fix the indents

**Comment by wking** [comment]
9208274f1 looks good to me :).

**Comment by vbatts** [comment]
LGTM

**Comment by jonboulle** [comment]
lgtm

**Pull request merged**